pkg/localstore/localstore.go

```diff
@@ -278,7 +281,7 @@ func safeInit(rootPath, sharkyBasePath string, db *DB) error {
 // New returns a new DB.  All fields and indexes are initialized
 // and possible conflicts with schema from existing database is checked.
 // One goroutine for writing batches is created.
-func New(path string, baseKey []byte, ss storage.StateStorer, o *Options, logger log.Logger) (db *DB, err error) {
+func New(path string, baseKey []byte, ss storage.StateStorer, o *Options, logger log.Logger, validStamp postage.ValidStampFn) (db *DB, err error) {
```

Comment:
Can we add it to the Options and not an extra parameter? This way we dont need to add it everywhere?

Also, it can be initialized to nil and in the Sample function we can check if its not nil before using it. That way this doesnt have to be changed everywhere/

Reply:
And just to note: I do have one testnet node with one copy of my OSM tiles pinned from zooms 0 through 12 (vs mainnet zooms 0 through 18).  It has been successfully playing and winning the storage incentives, but I also know that it was wiped recently has doesn't have any expired stamp chunks in it, at least not pinned in my dataset.